docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+
+The following example shows a VoID Linkset in turtle notation with the minimum metadata given in the header. The metadata block links to the ChEMBL 17 RDF description and the UniProt March 2015 release. The `linkPredicate` tells us that the link is an exact match, i.e. the linked instances can be deemed equivalent for most applications, and the `linksetJustification` property states that the link is declared as a Database Cross Reference assertion, rather than being computed based on an equivalent protein sequence. These properties allow consuming applications to make more informed choices about their reuse of the data.
+
+```xml
+@prefix bdb: <http://vocabularies.bridgedb.org/ops#> .
+@prefix chembl: <http://rdf.ebi.ac.uk/resource/chembl/> .
+@prefix chembl_target_cmpt: <http://rdf.ebi.ac.uk/resource/chembl/targetcomponent/> .
+@prefix sio: <https://semanticscience.org/resource/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix uniprot: <http://purl.uniprot.org/uniprot/> .
+@prefix void: <http://rdfs.org/ns/void#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:chembl17-uniprot-exactMatch-linkset a void:Linkset ;
+    void:subjectsTarget :chembl17-rdf ;
+    void:objectsTarget <http://purl.uniprot.org/void#UniProtDataset_2015_03> ;
+    void:linkPredicate skos:exactMatch ;
+    bdb:linksetJustification sio:database-cross-reference
+    void:triples "6367"^^xsd:integer .
+
+chembl_target_cmpt:CHEMBL_TC_4803 skos:exactMatch uniprot:A0ZX81 .
+chembl_target_cmpt:CHEMBL_TC_2584 skos:exactMatch uniprot:A1ZA98 .
+...
+```
+
+
+#### Simple Standard for Sharing Ontology Mappings (SSSOM)
+
+The OBOFoundry Simple Standard for Sharing Ontology Mappings ([SSSOM](https://github.com/OBOFoundry/SSSOM)) is a newly emerging standard for exchanging mapping information with minimal metadata, although the minimal model is extensible. It consists of a tab-separated-value file (TSV) with each row representing a mapping. The columns in the file have been defined by the community ([latest version](https://github.com/OBOFoundry/SSSOM/blob/master/SSSOM.md)) and provide the mapping together with its provenance. At present, four columns are required (`subject_id`, `predicate_id`, `object_id`, `match_type`) with optional columns to provide more provenance to support the mapping, e.g. licensing information, author information, creation method. The use of CURIEs within the TSV is strongly encouraged.
+
+The following TSV shows our example data as a mapping file using the minimal columns (correct as of November 2020). The information provided is less than the minimal VoID model above.
+
+```
+subject_id    predicate_id    object_id    match_type
```

Comment:
Double check use of TABs, because they seem to be spaces

Reply:
Is this what you mean?

\```tsv
subject_id	predicate_id	object_id	match_type
chembl:CHEMBL_TC_4803	skos:exactMatch	uniprot:A0ZX81	sio:database-cross-reference
chembl:CHEMBL_TC_2584	skos:exactMatch	uniprot:A1ZA98	sio:database-cross-reference
\```

```tsv
subject_id	predicate_id	object_id	match_type
chembl:CHEMBL_TC_4803	skos:exactMatch	uniprot:A0ZX81	sio:database-cross-reference
chembl:CHEMBL_TC_2584	skos:exactMatch	uniprot:A1ZA98	sio:database-cross-reference
```